picard/ui/metadatabox/__init__.py

```diff
@@ -72,6 +71,7 @@
     TagDiff,
     TagStatus,
 )
+import dill
```

Comment:
This is an extra dependency, what is it needed for?

Reply:
Rather than writing a custom tag diff to json serialisation, I figured that it may be easier to just use something off the shelf.

Not sure it is worth the extra dependency though, unless it's useful anywhere else!

Reply:
Let's go without this dependency then. We are trying to avoid external dependencies if possible, or keep it optional. I think the use case here does not really warrant that. You need to build the data structure anyway, JSON seemed fine for serializing this.

Having this in JSON format means some other application could potentially implement support for this mimetype in clipboard easily (without needing to understand the internals of Python dill or pickle).